client/signup/site-mockup/site-mockup.jsx

```diff
@@ -0,0 +1,112 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import { isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Style dependencies
+ */
+import './site-mockup.scss';
```

Comment:
Should the contents of this file rather be included in `site-mockup/style.scss`? There was an effort to move all scss files to webpack in #27515, and the related work in signup was done in #28394, so that could require a naming convention to be followed?

Reply:
The notion here is just that the `SiteMockup` component should be considered separately from the way it is being implemented within the signup flow itself. This means that it probably just be moved into `client/components/site-mockup`